### PR TITLE
Add l'Équipe icon

### PR DESCRIPTION
**Pull request by Poukame**
lequipe

**Issue:** none
**Website:** https://lequipe.fr
**Similarweb rank:**
[Ranked #776](https://www.similarweb.com/website/lequipe.fr/#overview)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Picked the hex color from header on official website

### CODE REVIEW

**Review by adamrusted — CHANGES_REQUESTED** [review]
Some minor differences when comparing to [this SVG](https://commons.wikimedia.org/wiki/File:L%27%C3%89quipe_wordmark.svg), with your PR in black and the source file in red. Would you mind updating the source, and taking another shot at this one @Poukame? You may need to increase the precision of SVGO to 4 decimals in order to get it spot on.

image

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7282,6 +7282,11 @@
             "hex": "DF0522",
             "source": "https://uk.kyocera.com"
         },
+        {
+            "title": "L'Équipe",
+            "hex": "D61E00",
```
Comment (PratyushJayachandran) [inline _data/simple-icons.json:7287]:
```suggestion
            "hex": "E42829",
```
From the source logo, this was the color I saw.

**Review by PratyushJayachandran — COMMENTED** [review]
Logo matches perfectly @adamrusted 
Just the color seems different

**Review by adamrusted — APPROVED** [review]
Given @PratyushJayachandran's review of the icon, and update to the hex code, I'll get this merged in.

**Pull request merged**